keras_metrics/metrics.py

```diff
@@ -6,6 +6,11 @@
 def _int32(y_true, y_pred):
     y_true = K.cast(y_true, "int32")
     y_pred = K.cast(K.round(y_pred), "int32")
+    
+    if y_pred.shape[1] != 1:
+      y_pred = y_pred[:,1:2]
+      y_true = y_true[:,1:2]
```

Comment:
From what I understand, this code forces us to calculate the metrics only for a _second_ label.

Reply:
What if instead of making it explicitly during the conversion step, this parameter will be passed to the constructor of the metric itself, like:
```py
recall = keras_metrics.recall(label=2)
```

Reply:
discussed here https://github.com/netrack/keras-metrics/pull/5